pulp_smash/tests/rpm/cli/test_copy_units.py

```diff
@@ -181,3 +270,131 @@ def test_copy_langpacks(self):
+        # Choose the newest version of the given RPM.
+        rpm_version = self.rpm_dict.get(self.rpm_name, [None, None])[1]
+        self._copy_and_publish(self.rpm_name, rpm_version)
+        # Execute `yum update` on the RPM.
+        completed_proc = cli.Client(self.cfg).run(
+            'sudo yum update {}'.format(self.rpm_name).split()
+        )
+        # Check if the update succeeds; it should have updates.
+        self.assertNotIn('Nothing to do.', completed_proc.stdout.splitlines())
+
+    def _copy_and_publish(self, rpm_name, rpm_version):
+        """Copy the RPM with given name and version into 2nd repo and publish it.
+
+        :param rpm_name: The name of target module.
+        :param rpm_version: The version of target module.
+        """
+        # Copy the module from repo1 to repo2.
+        _copy_repo(self.cfg, _REPO_ID, self.repo_id, rpm_name, rpm_version)
+        # Search for the RPM's name/version in repo2's content.
+        self._search_rpm(self.repo_id, rpm_name, rpm_version)
+        # Publish repo2 and verify no errors.
+        completed_proc = cli.Client(self.cfg).run(
+            'pulp-admin rpm repo publish run --repo-id {}'
+            .format(self.repo_id).split()
+        )
+        phrase = 'Task Failed'
+        for stream in ('stdout', 'stderr'):
+            with self.subTest(stream=stream):
+                self.assertNotIn(phrase, getattr(completed_proc, stream))
+
+    def _search_rpm(self, repo_id, rpm_name=None, rpm_version=None):
+        """Search for a RPM in the repository with optional name or version.
+
+        :param repo_id: A RPM repository ID.
+        :param rpm_name: The name of target module.
+        :param rpm_version: The version of target module.
+        :returns: A :class: `pulp_smash.cli.CompletedProcess`.
+        """
+        completed_proc = cli.Client(self.cfg).run(
+            ' '.join([
+                'pulp-admin rpm repo content rpm --repo-id {}'.format(repo_id),
+                '--str-eq name={}'.format(rpm_name)
+                if rpm_name is not None else '',
+                '--str-eq version={}'.format(rpm_version)
+                if rpm_version is not None else ''
+            ]).split()
+        )
+        if rpm_name is None:
+            self.assertNotEqual('\x1b[0m', getattr(completed_proc, 'stdout'))
+        else:
+            self.assertIn(rpm_name, getattr(completed_proc, 'stdout'))
+            # Check if the version is matched iif rpm's name is found.
+            with self.subTest():
+                self.assertIn(rpm_version, getattr(completed_proc, 'stdout'))
+
+    @classmethod
+    def tearDownClass(cls):
+        """Delete the repositories and clean up orphans."""
+        super(CopyAndPublishTwoVersionsRepoTestCase, cls).tearDownClass()
+        # Delete the published repositories.
+        cli.Client(cls.cfg).run(
+            'sudo rm /etc/yum.repos.d/dev_pulp_repos_{}.repo'
```

Comment:
> 'sudo rm /etc/yum.repos.d/dev_pulp_repos_{}.repo'

Instead of this, it should be sudo rm `/etc/yum.repos.d/dev_pulp_repos_{}`**_**`.repo`. lol..Please let me execute the test case multiple times again to verify that.